pkg/platform/model/vulnerabilities.go

```diff
@@ -29,6 +29,15 @@ func (v Vulnerabilites) Length() int {
 	return len(v.Critical) + len(v.High) + len(v.Medium) + len(v.Low)
 }
 
+func (v *Vulnerabilites) Count() map[string]int {
+	return map[string]int{
+		"Critical": len(v.Critical),
+		"High":     len(v.High),
+		"Medium":   len(v.Medium),
+		"Low":      len(v.Low),
```

Comment:
I think these keys might need localization? Least the caps makes it seem these are intended for displaying.

Reply:
Not localization but definitely can use constants which have already been created.